client/my-sites/stats/wordads/highlights-section.jsx

```diff
@@ -15,6 +15,9 @@ function getAmountAsFormattedString( amount, stripZeros ) {
 }
 
 function useHighlights( earnings ) {
```

Comment:
IMHO It's very important to keep the interface consistent, rather than to simplify things in the mobile highlights code path, for the sake of easier adoption. If it's possible to handle the differences in one place rather than everywhere, then it's a win. Thoughts @dognose24?

Reply:
I'm not sure I understand your meaning here. Also related, some previous feedback from Dognose here:

https://github.com/Automattic/wp-calypso/pull/89360#pullrequestreview-1989758920

Are you recommending that I stick with sending along the `Icon` component and leave the SVG value out? What do you think about the hidden dependency in the `CountCard` component? I was thinking a good next step would be to update `CountCard` so it imports `Icon` and uses it directly.

It would be a small/quick change. I'm good with either approach though.

Reply:
I updated the `CountCard` to make the `Icon` dependency explicit. This also makes the interface consistent across mobile and Desktop.